plugins/tiddlywiki/tiddlyweb/config-tiddlers-filter.tid

```diff
@@ -1,2 +1,2 @@
-title: $:/config/Server/ExternalFilters/[all[tiddlers]] -[[$:/isEncrypted]] -[prefix[$:/temp/]] -[prefix[$:/status/]] -[[$:/boot/boot.js]] -[[$:/boot/bootprefix.js]] -[[$:/library/sjcl.js]] -[[$:/core]]
+title: $:/config/Server/ExternalFilters/[is[tiddler]] -[[$:/core]] -[[$:/library/sjcl.js]] -[prefix[$:/boot/]] -[prefix[$:/HistoryList]] -[status[pending]plugin-type[import]] -[[$:/isEncrypted]] -[prefix[$:/status/]] -[prefix[$:/state/]] -[prefix[$:/temp/]]
```

Comment:
IMO a filter should reduce the number of elements as fast as possible. I'm sure there are more `[prfix[xxx]]` elements than `$:/core` in the list. Why did you change the order in that way?

Reply:
See my comment here: https://github.com/Jermolene/TiddlyWiki5/pull/5384#issuecomment-758172726